_board/seeeduino_wio_terminal.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: download
+board_id: "seeeduino_wio_terminal"
+title: "Seeeduino Wio Terminal Download"
+name: "Seeeduino Wio Terminal"
+manufacturer: "SEEED"
+board_url: ""
+board_image: "seeeduino_wio_terminal.jpg"
+date_added: 2020-7-3
+---
+
+The Seeed 's Wio Terminal is an ATSAMD51-based microcontroller with wireless connectivity supported by Realtek RTL8720DN. Its CPU speed runs at 120MHz (Boost up to 200MHz). Realtek RTL8720DN chip supports both Bluetooth and Wi-Fi providing the backbone for IoT projects. The Wio Terminal itself is equipped with a 2.4” LCD Screen, onboard IMU(LIS3DHTR), microphone, buzzer, microSD card slot, light sensor, and infrared emitter(IR 940nm).
+
```

Comment:
I'm going to add a warning for now that the Bluetooth and WiFi isn't supported from CircuitPython. I don't want to mislead folks with what is supported. It'll be similar to the note here: https://github.com/adafruit/circuitpython-org/blob/master/_board/espressif_saola_1_wroom.md